crates/bevy_derive/src/modules.rs

```diff
@@ -34,24 +34,26 @@ impl Modules {
 }
 
 fn get_meta() -> Option<Modules> {
-    let manifest = Manifest::new().unwrap();
-    if let Some(package) = manifest.find(|name| name == "bevy") {
-        Some(Modules::meta(&package.name))
-    } else if let Some(package) = manifest.find(|name| name == "bevy_internal") {
-        Some(Modules::meta(&package.name))
-    } else {
-        None
-    }
+    Manifest::new()
+        .unwrap()
+        .find(|name| name == "bevy" || name == "bevy_internal")
```

Comment:
This makes the output of this function depend on the order of dependencies if both bevy and bevy_internal are specified as dependencies for some reason.

Reply:
yes true, though this was the previous behavior of the implementation IIUC?